src/index.html

```diff
@@ -5,9 +5,39 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="body">
+    <div class="card" data-qa="card">
+      <img src="./images/imac.jpeg" alt="Apple A1419"
+      class="card__image">
```

Comment:
Need to fix with code style and check it everywhere
```suggestion
      <img
        src="./images/imac.jpeg"
        alt="Apple A1419"
        class="card__image"
      >
```

Reply:
isn't fixed. Check this case and with a button as well